dependencies.txt

```diff
@@ -5,9 +5,8 @@ shapely
 networkx
 owslib
 map2model
-loopprojectfile==0.1.3
+loopprojectfile==0.1.4
 beartype
 gdal==3.8.4
```

Comment:
We should test this

Reply:
The pinned version will actually have to be 0.1.5, assuming that the PR in https://github.com/Loop3D/LoopProjectFile/pull/42 is released as a new version

Reply:
I meant the gdal requirement, not projectfile sorry

Reply:
The gdal requirement seems to be needed - see https://github.com/Loop3D/map2loop/pull/153/commits/b7c9c049393ba7eb0878be20f046413de5e18801 and subsequent commits. It's strange because it has worked in previous commits without a pin...